Code_Exercises/Exercise_01_Hello_SYCL/README.md

```diff
@@ -0,0 +1,24 @@
+# Exercise 1
+
+The purpose of this exercise is to compile some SYCL code.
+
+Write a simple code using the examples in `solution.cpp` and compile with
+```
+clang++ -fsycl solution.cpp
+```
+
+Try to run the code on different devices using 
+```
+SYCL_DEVICE_FILTER=somedevice ./a.out
```

Comment:
We should maybe say specifically `cuda:gpu` here.

Reply:
Same for some other other exercises.

Reply:
Change made